README.md

```diff
@@ -232,12 +232,13 @@ An object including the following properties:
 * `body` - the body
 * `params` - the params matching the URL
 * `req` - the incoming HTTP request from Node core
+* `log` - the logger instance of the incoming request
 
 <a name="reply"></a>
 #### Reply
 
 An object that exposes three APIs.
-* `.send(payload)` - Sends the payload to the user, could be a plain text, JSON, stream, or an Error object.
+* `.send(payload, [serializeSchema])` - Sends the payload to the user, could be a plain text, JSON, stream, or an Error object. *serializeSchema* if setted to `false`, disables the custom serializer for the current handler and uses the default one.
```

Comment:
I'm 👎  on this change, as I think overloaded methods generally adds complexity to Node.js code.
How about `reply.setSerializer(null)` or `reply.serialize(null)`? Then we can also support custom serializers, e.g. [msgpack5](https://github.com/mcollina/msgpack5)